src/Maestro/tests/Scenarios/default-channels.ps1

```diff
@@ -46,9 +47,40 @@ try {
     # Adding default channels'
     Write-Host "Creating default channels"
     try { Darc-Delete-Default-Channel -channelName $testChannel1Name -repoUri $repoUri -branch $branchName } catch {}
-    try { Darc-Delete-Default-Channel -channelName $testChannel2Name -repoUri $repoUri -branch $branchName } catch {}
+    try { Darc-Delete-Default-Channel -channelName $testChannel2Name -repoUri $repoUri -branch $branchNameWithRefsHeads } catch {}
     Darc-Add-Default-Channel -channelName $testChannel1Name -repoUri $repoUri -branch $branchName
-    Darc-Add-Default-Channel -channelName $testChannel2Name -repoUri $repoUri -branch $branchName
+    Darc-Add-Default-Channel -channelName $testChannel2Name -repoUri $repoUri -branch $branchNameWithRefsHeads
+
+    # Retrieve the default channel and ensure that the refs/heads section was removed,
+    # and ensure that when we query via the API, 'refs/heads' is appropriately stripped out.
+    # The publishing infrastructure uses the REST api, while darc uses a query of all default channels and then
+    # substring filters for a more flexible user experience.
+    $defaultChannels = Darc-Get-Default-Channel-From-Api -repoUri $repoUri -branch $branchName
+    if ($defaultChannels.Count -ne 2) {
+        throw "Unexpected number of default channels"
+    }
+    if (($defaultChannels[0].channel.name -ne $testChannel1Name) -and 
+        ($defaultChannels[1].channel.name -ne $testChannel1Name)) {
+        Write-Host $defaultChannels
```

Comment:
Was this left in for debugging, or do we want to also print the default channels in the other assert block?

Reply:
Just for debugging.  Removing